src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -729,7 +729,11 @@ export default class ResultsViewPage extends React.Component<
         if (
             this.urlWrapper.isPendingSession ||
             this.urlWrapper.isLoadingSession ||
-            !this.resultsViewPageStore.studies.isComplete
+            this.resultsViewPageStore.studies.isPending ||
+            (this.resultsViewPageStore.studies.isComplete &&
```

Comment:
could this just be simplified to 
```this.resultsViewPageStore.customDriverAnnotationReport
                    .isPending```

Reply:
No, this logic is something that was needed to get it working. I would like to bring to attention here that this mention of customDriverAnnotationReport at this place was needed to make sure that this data is prepared before the ResultsView loads. I think that the solution where would be rather to remove the need for this mechanism at this place, than to update the logic. Again, I am willing to update this, but I need input from your side on how to better solve this issue.